src/pip/_internal/cli/progress_bars.py

```diff
@@ -61,7 +65,8 @@ def _raw_progress_bar(
     iterable: Iterable[bytes],
     *,
     size: Optional[int],
-) -> Generator[bytes, None, None]:
+    chunk_size: Optional[int],
```

Comment:
This argument name seems a bit weird because `chunk_size` isn’t actually the size of the chunk (which is `len(chunk)`)

Reply:
Renamed it to "unit_size". if there's an alternative name you think would be better, please let me know